packages/tocco-ui/src/Icon/Icon.js

```diff
@@ -76,7 +81,17 @@ Icon.propTypes = {
   /**
    * Add spacing according position. Default value is 'none'. Possible values: after|before|between|none
    */
-  position: PropTypes.oneOf(['after', 'before', 'between', 'solely'])
+  position: PropTypes.oneOf([
+    stylingPosition.AFTER,
+    stylingPosition.BEFORE,
+    stylingPosition.BETWEEN,
+    stylingPosition.SOLELY]),
+
+  // TODO desribe props
```

Comment:
describe the props or remove the comment

Reply:
see 3dccf3f54eaabfd59b88d6fcecd19507df44d039